website/static/js/fangorn.js

```diff
@@ -115,10 +115,14 @@ function findByTempID(parent, tmpID) {
 
 function cancelUploads (row) {
     var tb = this;
-    var filesArr = tb.dropzone.getQueuedFiles();
+    var filesArr = tb.dropzone.getActiveFiles();
+    var rejectedFilesArr = tb.dropzone.getRejectedFiles();
     for (var i = 0; i < filesArr.length; i++) {
         var j = filesArr[i];
         if(!row){
+            if (j.status === 'uploading') {
```

Comment:
This seems suspect; `status` is set by Dropzone, correct? I'm not sure what the side effects of changing it here will be.